.github/workflows/parry-ci-build.yml

```diff
@@ -22,6 +22,18 @@ jobs:
       - uses: actions/checkout@v4
       - name: Clippy
         run: cargo clippy
+  doc:
+    runs-on: ubuntu-latest
+    env:
+      RUSTDOCFLAGS: -D warnings
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@v2
+        with:
+          prefix-key: ${{ env.RUST_CACHE_KEY }}
```

Comment:
I added that for consistency with rapier, but it might not be necessary 🤔 , I like the more straightforward approach of https://github.com/dimforge/parry/pull/227